syntax = "proto3";

option go_package = "pb/game";

import "proto/common/tip.proto";
import "proto/common/common.proto";
import "proto/common/empty.proto";

option cc_generic_services = true;

service PlayerClientCommonService 
{
  rpc SendTipToClient (TipInfoMessage) returns (Empty);
  rpc KickPlayer (TipInfoMessage) returns (Empty);
}